contrib/resources/translations/README.md

```diff
@@ -82,14 +84,24 @@ If you modify these strings, test the startup screen with `startup_verbosity = h
 for all the relevant `machine = (...)` settings. It is really easy to make a mistake
 here.
 
+### Country names
+
+For example `COUNTRY_NAME_USA` or `COUNTRY_NAME_BIH_LAT`. Usually the country code
+is taken from the _ISO 3166-1 alpha-3_ norm, but sometimes they can contain
+additional suffix specifying a language or alphabet, or a single 'country'
+can describe the whole geographic region, like in the real MS-DOS.
+
+After translating country names check them by executing DOSBox with `--list-countries`
```

Comment:
"the country names, chrck them by running DOSBox with the"

Reply:
OK, changed.